_includes/v22.2/known-limitations/cdc-transformations.md

```diff
@@ -0,0 +1,9 @@
+- It is necessary to pass the [`schema_change_policy='stop'`](create-changefeed.html#schema-policy) option in the changefeed creation statement when using the {% if page.name == "cdc-transformations.md" %} CDC transformations {% else %} [CDC transformations](cdc-transformations.html){% endif %} format. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/85143).
+- You can only apply CDC transformations on a single table in each statement.
+- Some stable functions, notably functions that return MVCC timestamps, are overridden to return the MVCC timestamp of the event.
```

Comment:
nit: we could link "stable functions" here: https://www.cockroachlabs.com/docs/stable/functions-and-operators.html#built-in-functions